packages/experiments/src/components/SelectedItemsList/SelectedItemsList.tsx

```diff
@@ -1,16 +1,19 @@
 import * as React from 'react';
 
 import { ISelectedItemsList, ISelectedItemsListProps, BaseSelectedItem } from './SelectedItemsList.types';
+
 const _SelectedItemsList = <TItem extends BaseSelectedItem>(
   props: ISelectedItemsListProps<TItem>,
   ref: React.Ref<ISelectedItemsList<TItem>>,
 ) => {
   const [items, setItems] = React.useState(props.selectedItems || props.defaultSelectedItems || []);
+  const { dragDropEvents, dragDropHelper } = props;
+
   const renderedItems = React.useMemo(() => items, [items]);
 
   React.useEffect(() => {
-    setItems(props.selectedItems || []);
-  }, [props.selectedItems]);
+    setItems(props.selectedItems || props.defaultSelectedItems || []);
+  }, [props.selectedItems, props.defaultSelectedItems]);
```

Comment:
why did we need to do this? i think selecteditems should just be dependent on defaultSelectedItems only on the first render.

Reply:
I did this because a test was failing (SelectedPeopleList.test.tsx, renders personas that are passed in as default). There were no items there.

I googled this. Turns out useEffects do run on initial render.  So looks like the items are set to default then immediately reset to whatever the selectedItems props is, which was 0. So this way they stay set to the default props. This may not be the best way to fix this. There is an alternate way to basically disable the effect for the first time. Maybe I should do that instead? But I figured that the defaultSelectedItems can change, so maybe we should be doing it this way? It's not really what was intended, but it is a possibility.

Reply:
I see, yea the reason we dont want to do this is - if someone changes the defaultSelectedItems, it would update the items. For the initial render, we do want the defaultSelectedItems to be set, but we dont have the items to be updated/dependent on a change in defaultSelectedItems. Hence we would not want it to be a part of useEffect.
Question-
- Has this unit test been failing? Or is it due to the changes we are doing in this PR?
If the unit test has been failing for sometime, i think it would make sense to fix the unit test. If not then we need to look into alternate ways of doing this.
I would want the selectedItems to be updated only when props.selectedItems is updated.

Reply:
Ok, updated it so that the effect doesn't run the first time.